blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+([scala-lang#1344](https://github.com/scala/scala-lang/pull/1344),
+[scala-lang#1348](https://github.com/scala/scala-lang/pull/1348),
+[scala-lang#1352](https://github.com/scala/scala-lang/pull/1352),
+[scala-lang#1373](https://github.com/scala/scala-lang/pull/1373),
+[docs.scala-lang#2388](https://github.com/scala/docs.scala-lang/pull/2388),
+[docs.scala-lang#2520](https://github.com/scala/docs.scala-lang/pull/2520)), improved the
+[contributing guide](https://github.com/scala/docs.scala-lang/pull/2350), and created an infrastructure to show both
+Scala 2 and Scala 3 versions of code examples in the documentation
+([docs.scala-lang#2450](https://github.com/scala/docs.scala-lang/pull/2450)). We created an
+[issue](https://github.com/scala/docs.scala-lang/issues/2481) to coordinate the update of the whole documentation to
+always show both Scala 2 and Scala 3 code examples when possible.
+
+**We offered individualized support to our MOOCs’ learners.** We published the course “Effective Programming in Scala”
+to the [Extension School platform](https://www.extensionschool.ch/learn/effective-programming-in-scala). This
+platform allows the learners to have regular 1-on-1 meetings with our instructors along their learning journey. Learn
+more about the motivation in the previous [announcement]({% link blog/_posts/2022-06-08-learn-scala-at-epfl-extension-school.md %}).
+
+### Developer Experience
+
+**We improved the debugger for Scala 3.** The Metals’ debugger is now able to evaluate arbitrary Scala 3 expressions
+while the debugger is paused in the middle of a program. Step-by-step execution now automatically skips the methods
+generated by the compiler (e.g. mix-in forwarders, getters, setters, bridge methods, and synthetic methods of case
+classes). Learn more in [this video](https://www.youtube.com/watch?v=UePrOa_1Am8).
+
+**We fixed the apps installed by the [setup instructions](https://docs.scala-lang.org/getting-started).** The
+command-line apps `sbt` and `scala` now behave consistently regardless of the installation process (`cs install` vs
+manual download of the release artifacts).
+
+### Community and Contributor Experience
+
+**We co-organized the [ScalaCon](http://www.scalacon.org/) online conference.** We were in charge of the program and
+the program committee.
+
+**We published videos about Scala 3.** [Let’s Talk About Scala 3](https://www.youtube.com/playlist?list=PLTx-VKTe8yLxYQfX_eGHCxaTuWvvG28Ml)
+is a series of videos where we share interesting, useful, and cool things related to Scala 3. We published 4 new
+videos in 2022.
+
+**We improved the code editing experience in [Scastie](https://scastie.scala-lang.org).** Scastie is the online Scala
+code editor that is used every day by one thousand developers to share code snippets in the community. We have 
+implemented “IDE features” such as autocompletion and showing type information under the mouse pointer.
```

Comment:
I'd suggest mentioning Metals here, so it's clear where these features are coming from. (it is Metals, isn't it?)

Reply:
OK, done in 6c333175